src/Tools/dotnet-monitor/HostBuilderHelper.cs

```diff
@@ -130,6 +130,10 @@ public static IHostBuilder CreateHostBuilder(HostBuilderSettings settings)
                             options,
                             urls,
                             metricsOptions.Enabled.GetValueOrDefault(MetricsOptionsDefaults.Enabled) ? metricUrls : Array.Empty<string>());
+
+                        // Since the endpoints have already been defined on the KestrelServerOptions, clear out the urls
+                        // so that the address binder does not log a warning about overriding urls as specified by configuration.
+                        context.Configuration[WebHostDefaults.ServerUrlsKey] = string.Empty;
```

Comment:
Have you tested this with dotnet config show? I'm assuming it will still work correctly.

Reply:
This appears to work correctly:

```
dotnet dotnet-monitor.dll config show --urls "http://localhost:52327;http://+:52329"

{
  "urls": "http://localhost:52327;http://\u002B:52329",
   ...
}
```